ledger-api/grpc-definitions/com/daml/ledger/api/v1/admin/party_management_service.proto

```diff
@@ -188,7 +188,7 @@ message PartyDetails {
   // Optional
   string display_name = 2;
 
-  // true if party is hosted by the participant.
+  // true if party is hosted by the participant and party is within the user's identity provider.
```

Comment:
`within` means different things if you are an admin vs. non-admin. Can expand on that?

Reply:
it is actually not different. a participant_admin will see all parties which are not assigned to default idp as non-local

Reply:
@mziolekda I would appreciate your help on documenting this one...

Reply:
Maybe rephrase to say:
```suggestion
  // true if party is hosted by the participant and the party shares the same identity provider as the user issuing the request.
```